### DISCUSSION TITLE
baseline bot の実行時エラーを解消する

### DISCUSSION THREAD

**Discussion by smly** [smly/mjai.app#2] [Ideas]
## 背景

mjx の bot 実装例などが共有されているが、いくつかの試合では実行時エラーによってペナルティを受けている。この状態を解消する。

## AC

* mjx については `gateway.py` のバグによるエラーを解消する
* 役なしで和了しようとする場合があるため、原因を明らかにする。`gateway.py` で対処できる範囲内で問題を解消する
* 他家の打牌時にポンなどの選択を行う場面で打牌しようとするため。原因を明らかにしてい問題を解消する

**Comment by ktsujino** [comment]
mjxを使って簡易なルールベースのbotを作ったのですが、同じ問題の影響を受けているようです。
参加ゲームの10%程度でエラー終了となっています。
